diff --git a/MODULE.bazel b/MODULE.bazel
index f5c16706..773a8c62 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -159,13 +159,13 @@ use_repo(
     "rules_rust_wasm_bindgen_cli",
 )
 
-rust = use_extension("//rust:extensions.bzl", "rust")
-rust.toolchain(edition = "2021")
-use_repo(rust, "rust_toolchains")
+# rust = use_extension("//rust:extensions.bzl", "rust")
+# rust.toolchain(edition = "2021")
+# use_repo(rust, "rust_toolchains")
 
-register_toolchains(
-    "@rust_toolchains//:all",
-)
+# register_toolchains(
+#     "@rust_toolchains//:all",
+# )
 
 register_toolchains(
     "//proto/protobuf:default-proto-toolchain",
