diff --git a/rust/platform/triple_mappings.bzl b/rust/platform/triple_mappings.bzl
index 73e89467..fcfe5f26 100644
--- a/rust/platform/triple_mappings.bzl
+++ b/rust/platform/triple_mappings.bzl
@@ -54,6 +54,7 @@ SUPPORTED_T2_PLATFORM_TRIPLES = {
     "i686-unknown-freebsd": _support(std = True, host_tools = False),
     "powerpc-unknown-linux-gnu": _support(std = True, host_tools = True),
     "riscv32imc-unknown-none-elf": _support(std = True, host_tools = False),
+    "riscv32imac-unknown-none-elf": _support(std = True, host_tools = False),
     "riscv64gc-unknown-none-elf": _support(std = True, host_tools = False),
     "s390x-unknown-linux-gnu": _support(std = True, host_tools = True),
     "thumbv6m-none-eabi": _support(std = False, host_tools = False),
@@ -118,6 +119,7 @@ _CPU_ARCH_TO_BUILTIN_PLAT_SUFFIX = {
     "powerpc64le": None,
     "riscv32": "riscv32",
     "riscv32imc": "riscv32",
+    "riscv32imac": "riscv32",
     "riscv64": "riscv64",
     "riscv64gc": "riscv64",
     "s390": None,

